crafting_pieces.xml

```diff
@@ -4758,10 +4758,10 @@
       <Material id="Iron6" count="2" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_empire_noble_sword_3_t5_blade" name="{=HwCYBWq2}Sharp Tipped Spatha Blade" tier="4" piece_type="Blade" mesh="empire_noble_blade_4" culture="Culture.empire" length="95.8" weight="1.2">
+  <CraftingPiece id="crpg_empire_noble_sword_3_t5_blade" name="{=HwCYBWq2}Sharp Tipped Spatha Blade" tier="4" piece_type="Blade" mesh="empire_noble_blade_4" culture="Culture.empire" length="97" weight=".97">

```

Comment:
lengths need to be edited in scale factor in weapon.xml. Do not edit length in crafting piece . Then length in crafting piece is set by the 3d modeller to fin the 3d model. Editing thje length here wil resulte in gaps between parts because of a difference between physical model and 3d model